python-sul-2017/videos/acessibilidade-para-deficientes-visuais-com-python.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Muito interessante a palestre e como o python pode agregar para diferentes áreas e aspectos\n\n",
+  "recorded": "09-10-2017",
```

Comment:
Please put the dates in YYYY-MM-DD format
https://github.com/pyvideo/data/blob/master/CONTRIBUTING.rst#json-objects
~~~ json
"recorded": "2017-10-09",
~~~
Review that the dates belong between the beginning and the end of the conference. (If you don't know, use the beginning date)